src/main/kotlin/io/github/mojira/arisa/domain/IssueUpdateContext.kt

```diff
@@ -0,0 +1,11 @@
+package io.github.mojira.arisa.domain
+
+import net.rcarz.jiraclient.Issue
+
+data class IssueUpdateContext(
+    val update: Issue.FluentUpdate,
+    val transition: Issue.FluentTransition,
+    var hasUpdates: Boolean = false,
+    var hasTransitions: Boolean = false,
+    var transitionName: String = "Update Issue"
```

Comment:
This currently does only support one transition.
It is conceivable that we will need both an update and another transition at some point though.

Reply:
Wdym?
The regular updates, yes.
But we have an update transition here. That's a custom transition on our tracker and has nothing to do with the normal update.

The normal update is just a put on the issue, while the update transition is something custom.

Reply:
Oh do you mean that you can add fields tro any transition?

You can only add fields to a transition that are available on the screen, otherwise you get a 400 (or something).

For resolve the screen only contains what's visible here:
![image](https://user-images.githubusercontent.com/12124394/82731307-25eee800-9d06-11ea-8f31-d3e62683b34d.png)

The update transition is the only one that supports these fields.

Reply:
Good to know!